examples/rest-grpc-multiplex/src/main.rs

```diff
@@ -0,0 +1,75 @@
+mod multiplex_service;
+
+use std::net::SocketAddr;
+use tonic::transport::Server;
+
+use rpc_helloworld::{greeter_server::Greeter, HelloReply, HelloRequest};
+use tonic::{Response as TonicResponse, Status};
+
+use axum::{response::Response, routing::get};
+use tower::ServiceExt;
+use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
+
+mod rpc_helloworld {
+    tonic::include_proto!("helloworld");
+}
+
+#[derive(Default)]
+struct GrpcServiceImpl {}
+
+#[tonic::async_trait]
+impl Greeter for GrpcServiceImpl {
+    async fn say_hello(
+        &self,
+        request: tonic::Request<HelloRequest>,
+    ) -> Result<TonicResponse<HelloReply>, Status> {
+        println!("Got a request from {:?}", request.remote_addr());
+
+        let reply = HelloReply {
+            message: format!("Hello {}!", request.into_inner().name),
+        };
+        Ok(TonicResponse::new(reply))
+    }
+}
+
+async fn web_root() -> &'static str {
+    "Hello, World!"
+}
+
+#[tokio::main]
+async fn main() {
+    // initialize tracing
+    tracing_subscriber::registry()
+        .with(tracing_subscriber::EnvFilter::new(
+            std::env::var("RUST_LOG")
+                .unwrap_or_else(|_| "example-rest-grpc-multiplex=debug".into()),
```

Comment:
Pretty sure this needs underscores.

```suggestion
                .unwrap_or_else(|_| "example_rest_grpc_multiplex=debug".into()),
```